interfaces/apparmor/template.go

```diff
@@ -362,3 +362,35 @@ var defaultTemplate = []byte(`
 ###SNIPPETS###
 }
 `)
+
+// classicTemplate contains apparmor template used for snaps with classic confinement.
+//
+// It can be overridden for testing using MockClassicTemplate().
+// Designed by jdstrand https://github.com/snapcore/snapd/pull/2366#discussion_r90101320
+var classicTemplate = []byte(`
+#include <tunables/global>
+
+###VAR###
+
+###PROFILEATTACH### (attach_disconnected) {
```

Comment:
This would be at odds with the rest of the code that adds "complain" but only if JailMode is not specified.